### PR TITLE
Fix SwiftPM support on iOS now that SPM supports resources.

### PR DESCRIPTION
**Pull request by noremac**
Still need to do some testing but I think this should work. Putting it up sooner than later to get some feedback. I did not manually include the storyboards because SPM can figure those out on its own.

## To be done
- Verify localized string stuff
- Probably more work to do on mac to not include the storyboards..

### CODE REVIEW

File: Sources/Extensions.swift [hunk 1]
```diff
@@ -11,7 +11,11 @@ import Foundation
 internal extension String {
 
     var lt_localized: String {
+        #if SWIFT_MODULE
```
Comment (krzysztofzablocki) [inline Sources/Extensions.swift:14]: can't use `#if SWIFT_PACKAGE` ?

Comment (noremac) [reply Sources/Extensions.swift]: Oh didn't realize that came for free, amended.

**Comment by oskarko** [comment]
@krzysztofzablocki when do you think you will be able to merge this PR?

**Comment by krzysztofzablocki** [comment]
@oskarko I think it still needs testing? I don't have time now, but if you can help with that then I can merge it once it's verified

**Comment by oskarko** [comment]
Doesn't seem to work either. Still '**Could not find a storyboard named**' issue.

**Comment by haemi** [comment]
@oskarko any updates on this?

**Comment by oskarko** [comment]
> @oskarko any updates on this?

I gave up with this branch, mate.

**Comment by krzysztofzablocki** [comment]
@oskarko @haemi you could just take @noremac work and finish it up, it shouldn't be a big lift
feel free to make separate PR

**Pull request closed without merging**